karapace/config.py

```diff
@@ -40,6 +41,7 @@
     "registry_port": 8081,
     "registry_ca": None,
     "log_level": "DEBUG",
+    "log_format": "%(name)-20s\t%(threadName)s\t%(levelname)-8s\t%(message)s",
```

Comment:
This is preparation work, another PR uses this to configure Karapace's logs.

Reply:
This is used here:

https://github.com/aiven/karapace/pull/391/files#diff-c87c5173e88bb0f60b00107f4586304f4869a976a4f998425efe3481c0db9f56R108-R111

It is useful for our CI, since we don't have systemd-journald adding the timestamps in that case.